app/assets/stylesheets/components/_deploy.scss

```diff
@@ -0,0 +1,19 @@
+.deploy-hidden {
+  display: none;
+}
+
+.deploy-box {
+  min-height: 200px;
+}
+
+.deploy-bg-loading {
+  background: $gray-light;
+}
+
+.deploy-bg-error {
```

Comment:
we already have classes that have the same effect (`bg-red`) -- can you use those?